drivers/sqljs-driver/build.gradle

```diff
@@ -0,0 +1,79 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
```

Comment:
Is there any reason to use multiplatform in this module instead of only the js plugin?

Reply:
Hmm no, I just switched to the js plugin